notebooks/validation_notebooks/slsim_dc2_diffsky_galaxy_distribution_comparision.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<p>is there a possibility to produce these histograms also straight out of SLSim? That would be nice to see what is really been done in SLSim and if things change, then we have checks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Hi @sibirrer , I created this plot directly from SLSim!